src/storage/DataAccessorBasedStore.ts

```diff
@@ -97,6 +105,9 @@ export class DataAccessorBasedStore implements ResourceStore {
   public async getRepresentation(identifier: ResourceIdentifier): Promise<Representation> {
     this.validateIdentifier(identifier);
 
+    if (this.metaStrategy.isAuxiliaryIdentifier(identifier)) {
+      return this.getMetadata(identifier);
```

Comment:
I am not sure if you want the containment triples in the .meta resource.

Reply:
I do. It's part of the metadata of a container. Otherwise we end up in a weird situation where `ldp:contains` is part of the internal metadata when you get the resource itself, but not if you only get its metadata.